### DISCUSSION TITLE
Azzeramento valori Tasmota

### DISCUSSION THREAD

**Discussion by Newreader** [indomus/forum#1959] [GENERALE - Discussioni generiche]
Salve, ho utilizzato una presa flashata con Tasmota per misurare i consumi di un dispositivo. Ora vorrei spostare la stessa presa su un altro dispositivo quindi vorrei azzerare tutte le grandezze legate all'energia memorizzate, ovvero vorrei fare un reset dei valori memorizzati e ripartire da zero. Ho cercato un comando Tasmota che facesse al caso mio, ma non l'ho ancora trovato.
Sapete indicarmi quale comando dare o la procedura da seguire per ottenere questo azzeramento?

**Comment by snakuzzo** [comment]
https://tasmota.github.io/docs/Commands/#power-monitoring

![image](https://user-images.githubusercontent.com/7205371/193642084-398a61c0-4734-4a61-a33c-4cde006e8165.png)

**Reply by snakuzzo** [reply]
appena testato su una mia presa...

```
18:37:28.655 CMD: energytotal
18:37:28.662 MQT: stat/tasmota-forno/RESULT = {"EnergyTotal":{"Total":364.554,"Yesterday":0.028,"Today":0.018}}
18:37:32.510 CMD: energytotal 0
18:37:32.518 MQT: stat/tasmota-forno/RESULT = {"EnergyTotal":{"Total":0.018,"Yesterday":0.028,"Today":0.018}}
```

**Reply by snakuzzo** [reply]
![image](https://user-images.githubusercontent.com/7205371/193642754-b762c69f-dabc-47fb-9fee-fd6bf3b78c4c.png)

**Comment by Newreader** [comment]
Grazie, come al solito sei sempre prezioso. Anch'io nelle mia ricerca ho trovato lo stesso risultato, difatti ho azzerato dando da console il comando:
cmnd/nomedispostivo/EnergyTotal 0
Grazie
